L.Control.Geonames.js

```diff
@@ -20,7 +21,7 @@ L.Control.Geonames = L.Control.extend({
         baseQuery: 'isNameRequired=true', //The core query sent to GeoNames, later combined with other parameters above
         position: 'topleft',
         showMarker: true, //Show a marker at the location the selected location
-        showPopup: true, //Show a tooltip at the selected location
+        showPopup: false, //Show a tooltip at the selected location
```

Comment:
We did not want to change the default property for this value.  Rather, we wanted to change the value passed in on the `mobileview.html` page.

Also, this is where you want to add the `alwaysOpen` option and set it to false by default, instead of as a hidden property above `_alwaysOpen`.